src/SFA.DAS.RequestApprenticeTraining.Domain/Interfaces/IEmployerRequestEntityContext.cs

```diff
@@ -38,7 +38,10 @@ public async Task<EmployerRequest> GetFirstOrDefault()
         public async Task<List<AggregatedEmployerRequest>> GetAggregatedEmployerRequests(long ukprn)
         {
             var result = await Entities
-                .Where(er => er.RequestStatus == Models.Enums.RequestStatus.Active)
+                .Where(er => er.RequestStatus == Models.Enums.RequestStatus.Active
```

Comment:
This is out of sync somehow - would not expect this to be here - should have been added in some other ticket

Reply:
It was a bug I noticed while testing this story, where if you respond to a request which was created > 12 months ago it was still appearing on the AggregatedRequests page but when you get to the select page it could not be viewed any more.  i will move the fix to QF2068 (Confirmation page) as it probably belongs there

Reply:
removed and i will fix the logic of the changes on the correct branch once Dil has confirmed what happens with Active/cancelled/Expired requests